week-2/L-array-methods-2/exercise3.js

```diff
@@ -6,8 +6,12 @@
 
 var ukNations = ["Scotland", "Wales", "England", "Northern Ireland"];
 
+
+function isInUK(country) {
+  return ukNations.includes(country); // complete this statement
```

Comment:
This is the right line of code, but I think this code doesn't run. Can you fix it?